### PR TITLE
Improve LLM Backbone Explanations 

### PR DESCRIPTION
**Pull request by mtanco**
Addresses #765 by updating the documentation and tool tip explaining how the LLM Backbone parameter works in the UI.

### CODE REVIEW

File: documentation/docs/tooltips/experiments/_llm-backbone.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 The **LLM Backbone** option is the most important setting as it sets the pretrained model weights.
 
-- Usually, it is good to use smaller architectures for quicker experiments and larger models when aiming for the highest accuracy
-- If possible, leverage backbones pre-trained closely to your use case
-- Any huggingface model can be used here (not limited to the ones in the dropdown list)
\ No newline at end of file
+- Use smaller models for quicker experiments and larger models for higher accuracy
+- Aim to leverage models pre-trained on tasks similar to your use case when possible
+- Select a model from the dropdown list or type in the name of a model on HuggingFace of your preference
```
Comment (pascal-pfeiffer) [inline documentation/docs/tooltips/experiments/_llm-backbone.mdx:5]:
```suggestion
- Select a model from the dropdown list or type in the name of a model on Hugging Face of your preference
```

File: documentation/docs/guide/experiments/create-an-experiment.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Follow the relevant steps below to create an experiment in H2O LLM Studio.
 4. Provide a meaningful **Experiment name**.
 
 5. Define the parameters. The most important parameters are:
-    - **LLM Backbone**: This parameter determines the LLM architecture to use. It is the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended types of foundation models, but you can also use [Hugging Face models](https://huggingface.co/models).
+    - **LLM Backbone**: This parameter determines the LLM architecture to use. It is the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended foundation models available in the dropdown list. You can also type in the name of a [HuggingFace model](https://huggingface.co/models) that is not in the list, for example: `Qwen/Qwen1.5-4B-Chat` or the name of a local folder that has the model you would like to fine-tune.
```
Comment (pascal-pfeiffer) [inline documentation/docs/guide/experiments/create-an-experiment.md:19]:
```suggestion
    - **LLM Backbone**: This parameter determines the LLM architecture to use. It is the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended foundation models available in the dropdown list. You can also type in the name of a [HuggingFace model](https://huggingface.co/models) that is not in the list, for example: `h2oai/h2o-danube2-1.8b-sft` or the path of a local folder that has the model you would like to fine-tune.
```

**Review by pascal-pfeiffer — APPROVED** [review]
Thank you @mtanco !
Two nitpicks in the comments

**Review by sherenem — APPROVED** [review]
LGTM, thank you @mtanco !

File: documentation/docs/guide/experiments/create-an-experiment.md (hunk 2 above)
Comment (sherenem) [inline documentation/docs/guide/experiments/create-an-experiment.md:19]:
```suggestion
    - **LLM Backbone**: This parameter determines the LLM architecture to use. It is the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended foundation models available in the dropdown list. You can also type in the name of a [Hugging Face model](https://huggingface.co/models) that is not in the list, for example: `Qwen/Qwen1.5-4B-Chat` or the name of a local folder that has the model you would like to fine-tune.
```

File: documentation/docs/tooltips/experiments/_llm-backbone.mdx (hunk 1 above)
Comment (sherenem) [inline documentation/docs/tooltips/experiments/_llm-backbone.mdx:5]:
```suggestion
- Select a model from the dropdown list or type in the name of a Hugging Face model of your preference
```

**Pull request merged**